src/components/Course.css

```diff
@@ -0,0 +1,16 @@
+div#Course {
```

Comment:
Course is used in a large project so consider prefixing it (e.g., #course-container) to avoid potential conflicts with other IDs.